getdata_HN.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+import json
+import os.path
+import requests
+from urllib.parse import urljoin
+
+URL = 'https://hacker-news.firebaseio.com/v0/'
+KEYWORDS = ['python', 'django']
+DATAFILE = 'data.json'
+stories = {}
+
+
+def _get_json(param):
+    url = urljoin(URL, param)
+    res = requests.get(url)
+    return res.json()
+
+
+def _get_stories():
+    parameter = 'topstories.json?print=pretty'
+    return _get_json(parameter)
+    '''
+    parameter = 'topstories.json?print=pretty'
+    story_ids = []
+    for story_id in _get_json(parameter):
+        if len(story_ids) == 150:
+            break
+        else:
+            story_ids.append(story_id)
+    return story_ids
+    '''
+
+
+def _get_stories_id():
+    stories_id = []
+    for story in _get_stories():
+        parameter = 'item/{}.json?print=pretty'.format(story)
+        story = _get_json(parameter)
+        for kwd in KEYWORDS:
+            if (kwd in story['title']
+                ) or (kwd.upper() in story['title']
+                      ) or (kwd.capitalize() in story['title']):
+                stories_id.append(story['id'])
+    return stories_id
```

Comment:
why return only ids? when you already get url and title. Just store them in the dict and you will not need to re-request them afterward